services/botVerificationService.js

```diff
@@ -0,0 +1,26 @@
+const jwt = require("jsonwebtoken");
+const { CLOUDFLARE_WORKER } = require("../constants/bot");
+
+/**
+ * Generates the JWT
+ *
+ * @param payload {Object} - Payload to be added in the JWT
+ * @return {String} - Generated JWT
+ */
+const generateToken = () => {
```

Comment:
why are we generating a token here, token will be generated in Cloudflare worker right?

Reply:
Yes but for testing purposes we require a util which can be used anywhere in the testing part. Apart from that it's not used anywhere in the actual code

Reply:
cant we directly write it as a function in test fixtures?

Not sure if it is a good approach please ask others about it as well

Reply:
We can create a util in testing helpers

Reply:
It's done